prometeo/cgen/code_gen_c.py

```diff
@@ -39,6 +39,7 @@
         'pmt_gemm_tn': 'c_pmt_gemm_tn', \
         'pmt_gemm_nt': 'c_pmt_gemm_nt', \
         'pmt_trmm_rlnn': 'c_pmt_trmm_rlnn', \
+        'pmt_syrk_ln': 'c_pmt_syrk_ln', \
         'pmt_gead': 'c_pmt_gead', \
         'pmt_getrf': 'c_pmt_getrf', \
         'pmt_getrsm': 'c_pmt_getrsm', \
```

Comment:
what is `getrsm` ?

Reply:
GEneral TRiangular Solve with Matrix right hand-side. I guess we can drop it and have just `getrs`? It's also a matter of type of the arguments and how overloading is handled by prometeo's parser. For now I found it easier to have two separate methods `getrsm` and `getrsv` (the second is not even implemented atm).